api/server/server.go

```diff
@@ -1206,7 +1220,10 @@ func postBuild(eng *engine.Engine, version version.Version, w http.ResponseWrite
 		authConfig        = &registry.AuthConfig{}
 		configFileEncoded = r.Header.Get("X-Registry-Config")
 		configFile        = &registry.ConfigFile{}
-		job               = eng.Job("build")
+		buildConfig       = builder.NewBuildConfig()
+		stdout            = engine.NewOutput()
```

Comment:
I think it's used a few lines below, but I don't know if that's even necessary anymore:

https://github.com/docker/docker/pull/12457/files#diff-44a527ea72ea9572e1cf71c7f1e2a0f0R1251